tasks/2017/submit-a-pull-request.md

```diff
@@ -44,16 +46,18 @@ if you are not sure how to do this.
 
 ## Submit
 
-* Open a Pull Request in the zulip-gci repository with title "Submit a Pull Request".
-  Refer to [this tutorial](https://guides.github.com/activities/hello-world/#pr)
+* Open a Pull Request in the [zulip-gci-submissions](
+  https://github.com/zulip/zulip-gci-submissions) repository with title
+  "Submit a Pull Request". Refer to [this tutorial](
+  https://guides.github.com/activities/hello-world/#pr)
   if you don't know how to open a pull request.
 
 * Sign the Dropbox CLA using the link posted by the Dropbox CLA bot.
 
 * Post a link to the pull request when you submit your task on the GCI
   website. The link should look like
-  `https://github.com/zulip/zulip-gci/pull/<number>`, where `<number>` is a 2 or
-  3 digit number.
+  `https://github.com/zulip/zulip-gci-submissions/pull/<number>`, where
+  `<number>` is a 2 or 3 digit number.
```

Comment:
This should be like 1 to 3 digit number.

Reply:
yeah, I think it will be 2 to 3 digit number pretty soon, but still technically we're starting from 3 at the lowest :)